SPECS/selinux-policy/0040-For-systemd-hostnamed-service-to-run.patch

```diff
@@ -0,0 +1,64 @@
+From 8979659f697ee472a0cea42018cd5d9d46066c45 Mon Sep 17 00:00:00 2001
```

Comment:
wrong commit id, the correct one is 08866e625320c86b5914a6e4fd4aee794cf30b8d, i.e. https://github.com/SELinuxProject/refpolicy/commit/08866e625320c86b5914a6e4fd4aee794cf30b8d